README.md

```diff
@@ -175,7 +175,7 @@ ho = @hyperopt for resources=50, sampler=Hyperband(R=50, η=3, inner=RandomSampl
     minimum(res), get_state(res) # return the minimum value and a state from which to continue the optimization
 end
 ```
-you may decide how to interpret the amount of resources, as a time limit, maximum number of iterations etc. 
+you may decide how to interpret the amount of resources as a time limit, maximum number of iterations etc. 
```

Comment:
```suggestion
you may decide how to interpret the amount of resources, as a time limit, maximum number of iterations etc. 
```

Reply:
I'm pretty sure the comma has to be there for the sentence to be readable. It could potentially be replaced by a colon

Reply:
How about:
"the resources are increased by defining a variable `resources` inside each loop, which grows according to the hyperband algorithm. What how to interpret `resources` is entirely up to the user - it can be a time-limit, the maximum number of iterations, or anything else."

Reply:
"the resources are increased by defining a variable `resources` inside each loop, which grows according to the hyperband algorithm. How to interpret `resources` is entirely up to the user - it can be a time limit, the maximum number of iterations, or anything else."

Reply:
I think that is extremely clear ^_^

Since it seems we are in agreement, I will copy-paste your latest suggestion and resolve this conversation.